web/packages/extension/src/utils.ts

```diff
@@ -11,17 +11,16 @@ const DEFAULT_OPTIONS: Options = {
 export let i18n: {
     getMessage(name: string): string;
 };
+
+interface StorageArea {
+    get(keys?: string[]): Promise<Record<string, unknown>>;
+    remove(keys: string[]): Promise<void>;
+    set(items: Record<string, unknown>): Promise<void>;
```

Comment:
```suggestion
    get: (keys?: string[]) => Promise<Record<string, unknown>>;
    remove: (keys: string[]) => Promise<void>;
    set: (items: Record<string, unknown>) => Promise<void>;
```

Currently these are declared as methods. In TypeScript, methods are bivariant: assignments are allowed if they could be valid. For correctness, assigning them as functions is preferable: functions are [contravariant](https://github.com/microsoft/TypeScript/pull/18654) in strict mode, which more thoroughly assures correctness.

I hope this should catch the assignment on line 85, which is valid in practice but not in theory.